### ISSUE TITLE
Hyperbolic PDEs 

### ISSUE THREAD

**Issue by rruizdeaustri** [lululxvi/deepxde#393]
Hi Lu,

I was able to solve the issues I had to run deepdxe. Now I'm trying to use it for reproducing the results of paper: "Implicit Form Neural Network for Learning Scalar Hyperbolic Conservation Laws" by X. Zhang  et al. where the authors use a different definition of the loss function part related with the residuals compared with PINNs. So my question is whether is possible to modify the loss function in deepdxe  in a easy way (ie. without modifying the code). 

Thanks !!!
